superset/views/core.py

```diff
@@ -1974,7 +1974,8 @@ def _set_dash_metadata(dashboard, data):
             ):
                 slice_id = value.get("meta").get("chartId")
                 slice_ids.append(slice_id)
-                slice_id_to_name[slice_id] = value.get("meta").get("sliceName")
+                if "sliceName" in value.get("meta"):
+                    slice_id_to_name[slice_id] = value.get("meta").get("sliceName")
```

Comment:
Because on line #1970 and #1977 you know that both the `meta` and `sliceName` key exists I think it's clearer if you simply do, 

```
slice_id_to_name[slice_id] = value["meta"]["sliceName"]
```

Reply:
fixed.